src/coffee/order/CoffeeOrderBoard.java

```diff
@@ -0,0 +1,42 @@
+package coffee.order;
+
+import java.util.LinkedList;
+import java.util.List;
+
+
+
+
+public class CoffeeOrderBoard {
+    private List<Order> _orders;
```

Comment:
Underscore is not Java style. Pleasd, use Java naming conventions
[(https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html)](url)